vbfml/scripts/err_analysis

```diff
@@ -125,8 +151,8 @@ def predict(
     with open(f"{model_path}/features.pkl", "rb") as f:
         image_features = pickle.load(f)
 
-    # Validation sequence: Read the last 20% of events
-    read_range = (0.8, 1.0)
+    # Get the read range from the command line, default is (X,Y)
```

Comment:
Thanks for updating this! Could you update the comment to reflect that the default is `(0.8, 1.0)`?

Reply:
Sorry I missed that, I changed it